ui/components/multichain/import-tokens-modal/index.scss

```diff
@@ -11,17 +37,66 @@
     }
   }
 
+  &__modal-content {
+    padding-top: 200px;
+
+    @include screen-sm-max {
+      padding-top: 24px;
+    }
+  }
+
+  &__modal-dialog-content {
+    padding-left: 0 !important;
+    padding-right: 0 !important;
+    padding-top: 16px !important;
+  }
+
+
   &__autodetect {
     vertical-align: unset;
   }
 
   &__search-list {
+    -ms-overflow-style: none; /* for Internet Explorer, Edge */
+    scrollbar-width: none; /* for Firefox */
+    overflow-y: scroll;
     max-width: 100%;
     overflow: auto;
-    max-height: 200px;
+    max-height: 400px;
   }
 
+  &__active-tab {
+    color: var(--color-primary-default);
+  }
+
+  &__button-tab {
+    font-weight: 500;
```

Comment:
Is this necessary?  I don't think we've used this anywhere else in a long time.

Reply:
I believe it is necessary to make the button tabs font look like 
![image](https://github.com/MetaMask/metamask-extension/assets/10994169/110c1554-bc3b-4d37-a16d-21f212455c88)
 instead of 
 
![image](https://github.com/MetaMask/metamask-extension/assets/10994169/1e5b06b3-faa8-4cb2-855f-4c393bc0b33d)

Reply:
Looking at the home screen, I don't think we modify font-weight, so I think we can just remove it.

Reply:
Yes we had a discussion internally that the home screen does not have that font weight, we plan on having a separate PR to align that
Here is the link to the figma design https://www.figma.com/file/HKpPKij9V3TpsyMV1TpV7C/DS-Components?type=design&node-id=4701-19616&mode=dev

screenshot: 
![image](https://github.com/MetaMask/metamask-extension/assets/10994169/571743ba-1097-4d21-a0f5-80dec77a2433)